tests/test_templates_layers.py

```diff
@@ -548,11 +548,10 @@ def impr(self, request):
     def rots(self, request):
         return request.param
 
-    def test_random_layers_seed(self, n_layers, tol, seed):
-        """Test that pennylane.templates.layers.RandomLayers() gets deterministic when using seed."""
+    def test_random_layers_seed_deterministic(self, n_layers, tol, seed):
```

Comment:
worthwhile to use `pytest.mark.parameterize` with a couple different possible seed values?

Reply:
:+1: good idea

Reply:
The parametrised fixture "seed" should try different seeds [1,2,3] already. Or are you thinking of trying different seeds and their output value in a given circuit? 

Shall I rewrite the test or add another one?

Reply:
It is not clear to me what `mark.parametrize` does that a parametrised fixture does not...

Reply:
Sounds like you were already on the same page. I didn't realize that `seed` was already parameterized. :+1: